examples/download_reports.py

```diff
@@ -0,0 +1,57 @@
+import datetime
+import finite_state_sdk
+import requests
+
+
+def custom_download_report(token, organization_context, report_type="CSV", report_subtype="ALL_FINDINGS", asset_version_id=None, output_filename=None):
+    """
+    Demonstration of a method for getting a download URL.
+    Downloads a report from the Finite State Platform and saves it to the specified output_filename.
+    You could build your own method to do something else with the URL, or you can use the built-in finite_state_sdk.download_asset_version_report() method.
+    :param token: Finite State API token
+    :param organization_context: Finite State API organization context
+    :param report_type: The type of report to download. Valid values are "CSV" and "PDF"
+    :param report_subtype: The subtype of report to download. Valid values are "ALL_FINDINGS", "ALL_COMPONENTS", "EXPLOIT_INTELLIGENCE", and "RISK_SUMMARY". See API documentation for details.
+    :param asset_version_id: The asset version ID to download the SBOM for
+    :param output_filename: The filename to save the SBOM to
+    """
+    url = finite_state_sdk.generate_report_download_url(token, organization_context, asset_version_id=asset_version_id, report_type=report_type, report_subtype=report_subtype, verbose=True) -> str:
+
+    # Send an HTTP GET request to the URL
+    response = requests.get(url)
+
+    # Check if the request was successful (status code 200)
+    if response.status_code == 200:
+        # Open a local file in binary write mode and write the content to it
+        print("File downloaded successfully.")
+        with open(output_filename, 'wb') as file:
+            file.write(response.content)
+            print(f'Wrote file to {output_filename}')
+    else:
+        print("Failed to download the file. Status code:", response.status_code)
+
+
+def example_download_reports(token, organization_context):
```

Comment:
Similar to the previous comment regarding the use of enums in place of strings, applies here as well.

Reply:
Same here - will consider this for the next release